packages/fury-adapter-swagger/lib/json-schema.js

```diff
@@ -166,6 +166,15 @@ const convertSubSchema = (schema, references, swagger) => {
     actualSchema.type = 'string';
   }
 
+  if (schema.pattern && schema.minLength && schema.pattern.startsWith('^[') && schema.pattern.endsWith(']*$')) {
+    // If a schema has a minimal length (minLength) > 0 AND there is a regex
+    // such as: `^[A-z]*$`, the schema can resolve to an empty string which
+    // doesn't match minLength.
+    //
+    // JSON Schema Faker will fail in that case and get into an infinite loop.
+    actualSchema.pattern = schema.pattern.replace('*$', '+$');
```

Comment:
Is this replacing all occurrences of `*$`? From what I understand from this PR we should only modify the end.

Reply:
`$` refers to the end of the line in regex. So replacing `*$` (usually would happen once at the end), if it is in the schema multiple times I think it should be fine to replace it.